BinaryGap/Program.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace BinaryGap;
+
+public static class BinaryGapClass
+{
+    private static SolutionClass s_sol = new SolutionClass();
```

Comment:
```suggestion
    private static SolutionClass SolutionInstance = new SolutionClass();
```
like this without underscore, this isn't python :D  and change everywhere